misc_docs/syntax/decorator_taggedTemplate.mdx

```diff
@@ -0,0 +1,35 @@
+---
+id: "taggedTemplate-decorator"
+keywords: ["taggedTemplate", "tagged", "template", "decorator"]
+name: "@taggedTemplate"
+summary: "This is the `@taggedTemplate` decorator."
+category: "decorators"
+---
+**Since 11.1**
+
+The `@taggedTemplate` decorator is used to bind to JavaScript tag functions.
+
+### Example
+
+<CodeTab labels={["ReScript", "JS Output"]}>
+
+```res example
+type res 
```

Comment:
I think the name `res` is a bit confusing since it's pretty tied to ReScript. Maybe calling it `result` instead? Also, what _is_ the actual return type of that statement? Might be nice to add some sort of return type rather than an abstract type, just for clarity, even if it doesn't 100% match the actual return from Bun.